qt/aqt/data/web/css/BUILD.bazel

```diff
@@ -1,22 +1,22 @@
 load("@bazel_skylib//rules:copy_file.bzl", "copy_file")
 load("//ts:copy.bzl", "copy_files_into_group")
-load("compile_sass.bzl", "compile_sass")
+load("//ts:compile_sass.bzl", "compile_sass")
 
 compile_sass(
     srcs = glob(
         ["*.scss"],
         exclude = ["_*.scss"],
     ),
     group = "css_local",
+    deps = [
+        "//ts/sass:core_lib",
+        "//ts/sass:buttons_lib",
+        "//ts/sass:scrollbar_lib",
+        "//ts/sass:card_counts_lib",
+    ],
```

Comment:
The way you have done it is fine too, but just in case you weren't aware, I think we can also bundle multiple sass files into a single library, which may be a bit more convenient (but does mean changing one file will mean rebuilding anything that uses the library)